src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -39,4 +42,28 @@ public List<AdvertBasicInfoDTO> getByCategory(@PathVariable String category) {
         return advertService.getByCategory(category);
 
     }
+
+    @DeleteMapping("/advert/{id}")
+    public void deleteAdvert(@PathVariable Long id) {
+        advertService.deleteAdvertById(id);
+    }
+
+    @Transactional
+    @DeleteMapping("/advert/title/{title}")
+    public void deleteAdvertByTtile(@PathVariable String title) {
+        advertService.deleteAdvertByTitle(title);
+    }
+
+    @Transactional
+    @DeleteMapping("/adverts/user/{userId}")
+    public void deleteAdvertsByUserId(@PathVariable Long userId) {
+        advertService.deleteAdvertsByUserId(userId);
+    }
+
+    @Transactional
+    @DeleteMapping("/adverts/deleteByDescription")
```

Comment:
endpoint should not have verbs in mapping

Reply:
The endpoint was renamed